### PR TITLE
move mtxLora to analog namespace

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: src/utils/permutations.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const permutations = <T>( sequence: Array<T> ): Array<Array<T>> => {
```
Comment (DarkPark) [inline src/utils/permutations.ts:1]: would be nice to have some description with examples here

File: tests/analog/mtx.commands.collector.test.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import {IMtxCommand} from '../../src/analog/CommandBinaryBuffer.js';
+import MtxCommandsCollector from '../../src/analog/MtxCommandsCollector.js';
+import getBytesFromHex from '../../src/utils/getBytesFromHex.js';
+import permutations from '../../src/utils/permutations.js';
+
+
+const data = getBytesFromHex('00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f');
```
Comment (DarkPark) [inline tests/analog/mtx.commands.collector.test.ts:8]:
```suggestion
const collecedData = getBytesFromHex('00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f');
```

?

File: tests/analog/mtx.commands.sequence.test.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import * as Message from '../../src/analog/message.js';
+import * as uplinkCommands from '../../src/analog/commands/uplink/index.js';
+import MtxCommandsCollector from '../../src/analog/MtxCommandsCollector.js';
+import {MTXLORA} from '../../src/analog/constants/hardwareTypes.js';
+import {UPLINK} from '../../src/constants/directions.js';
+import permutations from '../../src/utils/permutations.js';
+import getHexFromBytes from '../../src/utils/getHexFromBytes.js';
+
+
+const expectedMtxBuffer = '4d1010796430280fff011d00000008001a00000008001d00000008011d00000008001a000000'
```
Comment (DarkPark) [inline tests/analog/mtx.commands.sequence.test.ts:11]:
```suggestion
const expectedMtxBuffer = `
    4d1010796430280fff011d00000008001a00000008001d00000008011d00000008001a000000
    08001d00000008011d00000008001a00000008001d00000008011d00000008001a0000000800
    1d00000008013a00000008013a00000008013a00000008013a000000080000
`;
```
?

File: src/obis-observer/index.ts [hunk 4]
```diff
@@ -1,4 +1,9 @@
+import UnknownCommand from './UnknownCommand.js';
+
 export * as commands from './commands/index.js';
 export * as constants from './constants/index.js';
 export * as message from './message.js';
 export * as utils from './utils/index.js';
+export {
```
[the remark this answers is not in the document]
Comment (Pnom) [reply src/obis-observer/index.ts]: fixed

File: tests/analog/mtx.commands.collector.test.ts (hunk 2 above)
Comment (Pnom) [reply tests/analog/mtx.commands.collector.test.ts]: fixed

File: tests/analog/mtx.commands.sequence.test.ts (hunk 3 above)
Comment (Pnom) [reply tests/analog/mtx.commands.sequence.test.ts]: fixed

File: src/utils/permutations.ts (hunk 1 above)
Comment (Pnom) [reply src/utils/permutations.ts:24]: fixed

File: src/utils/permutations.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ * //   [1, 2, 3],
+ * //   [1, 3, 2],
+ * //   [2, 1, 3],
+ * //   [2, 3, 1],
+ * //   [3, 1, 2],
+ * //   [3, 2, 1]
+ * // ]
+ * ```
+ */
+
```
Comment (DarkPark) [inline src/utils/permutations.ts:23]:
```suggestion
```

**Review by DarkPark — DISMISSED** [review]

**Review by DeadByElpy — DISMISSED** [review]

**Review by DarkPark — APPROVED** [review]

**Pull request merged**